src/plot_erpgrid.jl

```diff
@@ -13,6 +13,22 @@ Plot an ERP image.
     Electrode positions.
 - `ch_names::Vector{String}`\\
     Vector with channel names.
+- `hlines_grid_axis::NamedTuple = (;)`\\
```

Comment:
when do those appear?  are these due to the faceting? If yes, maybe add a word on that. If yes, will they also work if no facetting is used?

Reply:
they will appear on every axes